Pipfile

```diff
@@ -8,6 +8,7 @@ python_version = "3.7"
 
 [packages]
 jupyterlab = "==1.2.15"
+google-cloud-bigquery = "==1.25.0"
```

Comment:
You don't need to change this here. The Pipfile is responsible for setting up the overall requirements for development in any of the packages. In your specific package folder, your extensions `install_requires` section in the `setup.py` file dictates what your particular extension's dependencies are.